app/businesses/models.py

```diff
@@ -66,7 +73,7 @@ class StatusEnum(Enum):
     capacity = db.Column(db.Integer, nullable=True)
 
     business_hours = db.relationship('BusinessHour', backref='business', lazy=True)
-    address = db.relationship('Address', backref='business', lazy=True)
+    addresses = db.relationship('Address', backref='business', lazy=True)
```

Comment:
sur cette ligne, il y a pas d'Erreur je voulais faire un commentaire sur la ligne `70` (celle du status du business). Pourquoi pour la valeur par défaut du status tu n'utilise pas l'enum? Tu mets la valeur encore en claire. On voulais éviter ca c<est pour ca qu'on utilisent des enum. J'ai remarqué ca dans bcp d'endroit ou tu utilise un enum

Reply:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request